openmls_rust_crypto/Cargo.toml

```diff
@@ -17,8 +17,7 @@ sha2 = { version = "0.10" }
 aes-gcm = { version = "0.9" }
```

Comment:
Aes and chacha could be updated as well.
But that will need a few more changes. Not sure if you want to do that here.